galasa-ui/src/styles/test-runs/saved-queries/QueryItem.module.css

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+.sideNavItem {
+  display: flex;
+  gap: 0.5rem;
+  align-items: flex-start;
+  padding: 0.5rem 0;
+  position: relative;
+  transition: background-color 0.15s ease-in-out;
+}
+
+.sideNavItem.collapsed {
+  display: none !important;
+}
+
+.dragHandle,
+.starIcon {
+  cursor: grab;
+  visibility: hidden;
+  margin-left: 1rem;
+  flex-shrink: 0;
+  margin-top: 0.125rem;
+}
+
+.starIcon {
+  visibility: visible;
+  cursor: default;
+}
+
+.sideNavItem:hover {
+  background-color: var(--cds-layer-selected);
+}
+
+.sideNavItem:hover .dragHandle {
+  visibility: visible;
+}
+
+.dragging {
+  cursor: grabbing;
+  background-color: var(--cds-layer-selected);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+}
+
+.dragging .dragHandle {
+  visibility: visible;
+}
+
+.sideNavLink {
+  text-decoration: none;
+  color: var(--cds-text-primary) !important;
+  font-size: 0.875rem;
+  padding-right: 1rem;
+  flex-grow: 1;
+  outline: none;
+  white-space: normal;
+  word-break: break-word;
+  line-height: 1.4;
+}
+
+.sideNavLink:hover,
+.sideNavLink:focus,
+.sideNavLink:active {
+  text-decoration: none;
+  color: var(--cds-text-primary) !important;
+}
```

Comment:
Same here regarding `!important` - can these colours be set in a normal way without overriding anything?